core/src/main/java/org/frankframework/logging/IbisPatternLayout.java

```diff
@@ -61,6 +63,14 @@ public class IbisPatternLayout extends IbisMaskingLayout {
 		}
 	}
 
+	private static void initLogExpressionHiding(final Configuration config) {
+		String hideRegex = config.getStrSubstitutor().replace("${ff:log.hideRegex:-}");
+
+		if (hideRegex != null && !hideRegex.isBlank()) {
+			IbisMaskingLayout.addToGlobalReplace(hideRegex);
+		}
+	}
```

Comment:
This class is not completely self contained, it no longer requires a specific configuration (but it does require the new property lookup class).